orttraining/orttraining/python/training/ortmodule/_experimental/_load_config_from_json.py

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# _load_config_from_json.py
+
+import json
+import os
+from types import SimpleNamespace
+
+from onnxruntime.capi import _pybind_state as C
+from functools import reduce
+from .._graph_execution_manager import _SkipCheck
+from ..debug_options import DebugOptions, LogLevel
+
+def load_data_from_json(path):
+    """Loads data from the json file path provided."""
+
+    data = None
+    with open(path) as f:
+        data = json.load(f, object_hook=lambda d: SimpleNamespace(**d))
+
+    if data is None:
+        raise RuntimeError(f"No data found in provided json file {path}.")
+
+    return data
+
+def load_propagate_cast_ops(ortmodule_config_accessor, data):
+    """Load PropagateCastOps from json file onto ORTModule"""
+
+    propagate_cast_ops_str = 'PropagateCastOps'
+    if not hasattr(data, propagate_cast_ops_str):
+        return
+    print(f"Found keyword {propagate_cast_ops_str} in json. Loading attributes from file.")
```

Comment:
Is it for debugging only? If not, we probably need the Logger object to go through the proper verbosity

Reply:
Yeah, let me use a logger object. Since this was internal only, I considered it to be ok to print to stdout. But maybe using logger object would be better idea here.